### ISSUE TITLE
[BUG] - Accordion/Wordmark: focus state not working in Chrome / Edge 135

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#3827]
### JS Framework

all

### Porsche Design System version

3.27.3

### On which device(s) you can see the bug?

macOS / Windows

### In which browsers you can see the bug?

Chrome / Edge

### Browser version

135

### Reproduction Example

Storefront

### Steps to reproduce the behavior

1. Go to https://designsystem.porsche.com/v3/components/accordion/examples or https://designsystem.porsche.com/v3/components/wordmark/examples#link
2. Tab through accordion examples with the keyboard
3. See error: no focus state is visible

### Expected behavior

PDS focus outline should be visible

### Note

This issue seem to occur where the `:focus` outline is generated through CSS `:before` pseudo-class.
Official Chromium Bug Report: https://issues.chromium.org/issues/410743804

**Comment by o-hook** [comment]
Bug seems to be fixed with latest Chrome update 135.0.7049.96.
